packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -141,7 +143,52 @@ export const Alert: React.FunctionComponent<AlertProps> = ({
   const [timedOutAnimation, setTimedOutAnimation] = useState(true);
   const [isMouseOver, setIsMouseOver] = useState<boolean | undefined>();
   const [containsFocus, setContainsFocus] = useState<boolean | undefined>();
-  const dismissed = timedOut && timedOutAnimation && !isMouseOver && !containsFocus;
+  const shouldDismiss = timedOut && timedOutAnimation && !isMouseOver && !containsFocus;
+  const [isDismissed, setIsDismissed] = React.useState(false);
+  const { hasAnimations } = React.useContext(AlertGroupContext);
+  const { offstageRight } = alertGroupStyles.modifiers;
```

Comment:
Do we need to account for RTL with alert groups and the animations? Right now the full page demo with toast alerts doesn't seem to work at all when you flip on RTL (alerts do not pop up and the notification drawer doesn't open/close). Alerts do appear when you turn it off, so it's working but not displaying

Reply:
Great Question!

Reply:
https://github.com/patternfly/patternfly/issues/7365 to address RTL.